R/spark_compile.R

```diff
@@ -126,7 +126,7 @@ spark_compile <- function(name,
 #'
 #' @keywords internal
 #' @export
-spark_compile_package_jars <- function(package = rprojroot::find_package_root_file(),
+compile_package_jars <- function(package = rprojroot::find_package_root_file(),
                                        spark_versions = NULL,
                                        scalac = NULL,
```

Comment:
But if path_program is one of our functions and is unexported (which it
seems like it should be) then I'd say it's better to keep it as NULL

On Fri, Aug 5, 2016 at 9:54 PM, Kevin Ushey <notifications@github.com>
wrote:

> In R/spark_compile.R
> <https://github.com/rstudio/sparklyr/pull/150#discussion_r73778700>:
>
> > @@ -126,7 +126,7 @@ spark_compile <- function(name,
> >  #'
> >  #' @keywords internal
> >  #' @export
> > -spark_compile_package_jars <- function(package = rprojroot::find_package_root_file(),
> > +compile_package_jars <- function(package = rprojroot::find_package_root_file(),
> >                                         spark_versions = NULL,
> >                                         scalac = NULL,
>
> We try to find the program using path_program when this is NULL below,
> although perhaps we should just make the default path_program("scalac")
> instead of NULL.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/rstudio/sparklyr/pull/150/files/dafaaa2516e9a03fac9075f1222cb9285069e69a..26772d833cb691b37a37157f0c62a54a961be082#r73778700>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAGXx0Bb9YnIxSkx1SScazwxiUWjgZgHks5qc-lFgaJpZM4JeE4J>
> .
>